src/repository/opamDownload.ml

```diff
@@ -225,6 +225,7 @@ module SWHID = struct
 
   let log fmt = OpamConsole.log "CURL(SWHID)" fmt
 
+  let main_url = OpamUrl.of_string "http://softwareheritage.com"
```

Comment:
```
This domain name has been registered with Gandi.net
```

Reply:
wrong url indedd

Reply:
what about just sending a request such as `curl -X POST https://archive.softwareheritage.org/api/1`. It returns no output nor failure?

Reply:
In that way, we can put the check directly on fallback mechanism, not in `OpamUpdate`. :+1:

Reply:
even better, they have an api endpoint just for that: https://docs.softwareheritage.org/devel/apidoc/swh.web.api.views.ping.html?highlight=ping#get--api-1-ping-